librecores-ci/README.md

```diff
@@ -25,3 +26,49 @@ The [or1k Continuous Integration (CI) suite](https://github.com/openrisc/or1k-te
 Librecores-CI docker container in [Travis-CI](https://github.com/openrisc/mor1kx/blob/master/.travis.yml). 
 Parallel execution of tests runs in Librecores-CI docker environment. As a reference one can follow up the [blog](http://nancychauhan.in/stories/2019/06/08/gsoc-week1_2/) to get an idea of how librecores docker images can be 
 integrated to existing continuous integration suite. 
+
+#### Yosys Parser
+Yosys is a framework for Verilog RTL synthesis. For monitoring resource usages of any hardware project, this parser script (extract-yosys-stats.py) can help to visualise results better. This take input from huge yosys.log file and outputs 'Printing Statistics' like :
+```
+wire bits,16199
+public wires,1058
+public wire bits,11151
+memories,0
+memory bits,0
+processes,0
+cells,7310
+SB_CARRY,426
+SB_DFF,166
+SB_DFFE,885
+SB_DFFESR,569
+SB_DFFESS,13
+SB_DFFSR,56
+SB_DFFSS,2
+SB_LUT4,5182
+SB_RAM40_4K,11
+```
+
+#### Quick Start 
+
+To use Yosys Parser, you can see its implementation in <a href="https://github.com/openrisc/mor1kx">openrisc project<a>.
```

Comment:
Some grammer:
  - To use **the** yosys parser
  - you can see **it's**

However, I would just replace the sentence with:
  - You can see an example of this yosys parser script in the [mor1kx](https://github.com/openrisc/mor1kx) OpenRISC cpu Jenkins test suite.